File: 448._Find_All_Numbers_Disappeared_in_an_Array.py
class Solution:
    def findDisappearedNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        result=[]
        l=[0]*len(nums)
        for i in nums:
            l[i-1]=l[i-1]+1
        for i in range(len(l)):
            if l[i]==0:
                result.append(i+1)
        return result

File: 448._Find_All_Numbers_Disappeared_in_an_Array.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        vector<int> result;
        vector<int> a(nums.size(),0);
        // int a[nums.size()]={0};
        for(int i=0;i<nums.size();i++){
           a[nums[i]-1]++; 
        }
        for(int i=0;i<a.size();i++){
            if (a[i]==0)
                result.push_back(i+1);
        }
        return result;
    }
};